src/styles/components/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
+
+    @extend %bgc-image;
+
+    background-image: url('../images/star.svg');
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &.stars--4 > .stars__star:nth-child(-n + 4) {
+    background-image: url('../images/star-active.svg');
+  }
```

Comment:
The way you're applying the active star is not scalable. Right now, you're hardcoding the class for 4 active stars. What will happen if you need 1, 2, 3, or 5 active stars? You would have to write a new rule for each one. Instead, you could write a loop that generates these classes for you. Consider using a @for directive to create classes from stars--1 to stars--5.